Aeh/upd networking visuals

aidan-hubley opened the pull request:
**Changes**
- Unified visuals for:
    - Share circle page & search for circle page
    - lower navigation buttons across app

**Notes**
- Sought to benefit the user experience in 'networking' 
    - (when user A shows the 'share screen' to user B, the 'search screen' mirrors the visuals of the 'share screen'
 - Permission was given from @Statmosphere to redo the visuals off his new page, all previous functionality has remained untouched

**YouTrack Link**
[PC-153](https://prayercircle.youtrack.cloud/issue/PC-153/Circle-Networking-Visuals)

aidan-hubley committed:
Refactor share and settings buttons in
shareCircle.js

aidan-hubley committed:
Update circleSettings and shareCircle components

aidan-hubley committed:
formated join page to match share page

aidan-hubley committed:
uniform buttons

dev-arekusandoru committed:
update shareCircle presentation mode

dev-arekusandoru commented on app/(settings)/shareCircle.js:
```diff
@@ -31,13 +33,13 @@ export default function Page() {
 				</StyledView>
 
-				<StyledView className='absolute top-[20%] border-[10px] bg-offwhite border-purple p-[10px] rounded-xl'>
+				<StyledView className='absolute top-[15%] border-[10px] bg-offwhite border-purple p-[10px] rounded-xl'>
```
The percentage values here should really be changed for static px values

dev-arekusandoru commented on app/(settings)/shareCircle.js:
```diff
@@ -46,32 +48,47 @@ export default function Page() {
 					</StyledView>
 				</StyledView>
-				<StyledView className='absolute bottom-[25%] border-[10px] border-purple bg-white mt-20 p-[10px] rounded-xl'>
+				<StyledView className='absolute bottom-[30%] border-[10px] border-purple bg-white mt-20 p-[10px] rounded-xl'>
```
The borders on this page are really think and it doesn't flow with the design of any other pages. Im not opposed to the color but it looks out of place

dev-arekusandoru commented on app/joinCircle.js:
```diff
@@ -50,78 +54,78 @@ export default function Page() {
+						</StyledText>
 					</StyledView>
+
+					<StyledView className='absolute top-[15%] border-[10px] border-offwhite rounded-xl'>
```
The percentages for padding here are causing issues with how the page renders, at least on my device. It makes the page taller than the screen so the bottom buttons are hidden unless you scroll.

dev-arekusandoru commented on app/joinCircle.js:
```diff
@@ -50,78 +54,78 @@ export default function Page() {
+							/>
+						</StyledView>
+					</StyledView>
+					<StyledView className='absolute w-[70%] bottom-[30%] border-[10px] border-offwhite mt-20 rounded-xl'>
```
Percentage positioning here should go as well

dev-arekusandoru commented on app/joinCircle.js:
```diff
@@ -50,78 +54,78 @@ export default function Page() {
-						maxLength={8}
-						onChangeText={(text) => {
-							setCode(text);
+				<StyledView
```
This view needs to be outside of the scroll view for the buttons to be visible all the time, following the structure of all other pages

dev-arekusandoru reviewed (changes requested):
Few changes before this is ready for merge

aidan-hubley committed:
Update shareCircle styling

aidan-hubley committed:
Update profile page link to find circles

aidan-hubley committed:
Merge branch 'main' into AEH/upd_Networking_Visuals

aidan-hubley committed:
Add QR code and icon to shareCircle.js, fix
imports in findCircles.js, remove joinCircle.js,
and update button press in FilterItem.js

dev-arekusandoru committed:
fix safeAreaView error

dev-arekusandoru reviewed (approved):
Looks good.
